### DISCUSSION TITLE
Is it possible to import of connections programmatically?

### DISCUSSION THREAD

**Discussion by RckMrkr** [Sequel-Ace/Sequel-Ace#1301] [Q&A] [answered]
I am trying to improve my workflow and would like to know if it's programmatically possible to add connections to Sequel Ace.

Preferably all the information, but I gathered from the questions around import from Sequel Pro that the passwords may be difficult.

Is there a way, currently to accomplish this?

**Comment by Jason-Morcos** [comment, marked as answer]
You could try using mysql:// URLs and opening those in Sequel Ace? Those would pass most of the connection settings. Other than that, I don't know of an easy way to do this currently

**Comment by RckMrkr** [comment]
That does seem promising.
It will nok get me all the way there, but close enough.

Thank you.
